src/traces/isosurface/defaults.js

```diff
@@ -34,19 +34,49 @@ module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout
         return;
     }
 
+    var isomin = coerce('isomin');
+    var isomax = coerce('isomax');
+
+    if(isomin > isomax) {
```

Comment:
That's not what I meant. Please look at

https://github.com/plotly/plotly.js/blob/7d67f8011a0200cc09535cebde8de34199f85a09/src/components/colorscale/defaults.js#L33-L38

In brief, in `isomin > isomax` then both `isomin` and `isomax` should get coerced to their default value, that is `null`.

Reply:
Great comment!
Done in https://github.com/plotly/plotly.js/pull/3438/commits/bae7b56b6c7193f4be9213dee9847041cd503e69.
Thanks @etpinard

Reply:
Resolved. Thanks!